### PR TITLE
MST Creation

### PR DESCRIPTION
**Pull request by jctrotter**
Creates MST using Kruskal's, populates InputGraph mst and mst_distance variables with them.

NOTE: Kruskal's sorts by ascending edge weight, does this impact us in later steps?

### CODE REVIEW

File: mstEuclid.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
+import math
+from christofidesTSP import *
```
Comment (zshureih) [inline mstEuclid.py]: Isn't this a circular dependency?

**Review by zshureih — COMMENTED** [review]
Other than importing christofides into mstEuclid, I don't see anything wrong

File: mstEuclid.py (hunk 1 above)
Comment (jctrotter) [reply mstEuclid.py]: mm yeah maybe a little bit. I'll slap the function into christofides and commit back

Comment (zshureih) [reply mstEuclid.py]: Sounds good

**Pull request merged**